### PR TITLE
Change configurator messages to be only displayed on verbose mode

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: src/Configurator/AbstractConfigurator.php [hunk 1]
```diff
@@ -35,4 +35,15 @@ public function __construct(Composer $composer, IOInterface $io, Options $option
     abstract public function configure(Recipe $recipe, $config): void;
 
     abstract public function unconfigure(Recipe $recipe, $config): void;
+
+    protected function write($messages)
```
Comment (stof) [inline src/Configurator/AbstractConfigurator.php]: shouldn't it be a void-returning function ?

File: src/Configurator/AbstractConfigurator.php [hunk 2]
```diff
@@ -35,4 +35,15 @@ public function __construct(Composer $composer, IOInterface $io, Options $option
...
+    {
+        if (!is_array($messages)) {
+            $messages = [$messages];
+        }
+        foreach ($messages as $i => $message) {
+            $messages[$i] = '    '.$message;
+        }
+        return $this->io->write($messages, true, IOInterface::VERBOSE);
```
Comment (stof) [inline src/Configurator/AbstractConfigurator.php]: `IOInterface::write` does not return anything

File: src/Configurator/AbstractConfigurator.php (hunk 1 above)
Comment (fabpot) [inline src/Configurator/AbstractConfigurator.php]: done

File: src/Configurator/AbstractConfigurator.php (hunk 2 above)
Comment (fabpot) [inline src/Configurator/AbstractConfigurator.php]: indeed

**Pull request merged**